deploy/scripts/install-k8s.sh

```diff
@@ -32,6 +32,14 @@ else
   echo "yum not installed"
 fi
 
+#arch,suse distro condition
+if [ -x "$(command -v sealos)"];then
+  wget https://github.com/labring/sealos/releases/download/v4.1.3/sealos_4.1.3_linux_amd64.tar.gz \
+   && tar zxvf sealos_4.1.3_linux_amd64.tar.gz sealos && chmod +x sealos && mv sealos /usr/bin
```

Comment:
Does this script seem to work only in AMD64?

Reply:
okay,I will improve later

Reply:
I add arm support now!!!